### PR TITLE
doc: add minutes for meeting 20 Nov 2019

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2019-11-20.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* CPC 
+  * Focus is on onboarding
+    * https://github.com/openjs-foundation/project-onboarding/issues/1
+
+
+* Board
+  * meeting this week
+  * Only issue to bring to board is CommComm charter update FYI
```
Comment (Trott) [inline meetings/2019-11-20.md]: Sorry I missed the meeting, but the Build WG request isn't on tap yet? Or already happened? Or...?

File: meetings/2019-11-20.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+* N-API - preparing for workshop and 2.0 version of node-addon-api
+* Core promise APIs - massive PR that has been open for 6 months, is close to being ready to 
+  land. Will add experimental promise support for event handlers.  PR helps address common
+  problem that developers run into.  Please review/comment: https://github.com/nodejs/node/pull/27867
+* V8 Currency - nothing of note
+* Python 3 & Gyp - still some dusty corners, not all google tooling supports python 3
+* cmake - Michael mentioned this. Ben has been looking at it any may start working to push it
+  forward.  Want to make sure there is agreement that it is a reasonable way to move forward
+  and we had some discussion in the meeting around that. Please comment in the issue
+  If you have any concerns.https://github.com/nodejs/TSC/issues/648
```
Comment (Trott) [inline meetings/2019-11-20.md]:
```suggestion
  If you have any concerns. https://github.com/nodejs/TSC/issues/648
```

File: meetings/2019-11-20.md (hunk 1 above)
Comment (mhdawson) [reply meetings/2019-11-20.md]: Already went to the board, we were asked for a job role which was provided a few weeks ago and this week I'm going to present some slides with additional info/context about the build WG in the board meeting.

Comment (mhdawson) [reply meetings/2019-11-20.md]: Having said that I should add an issue to the board in admin to track that request

**Pull request merged**